docs/reference/docs/reindex.asciidoc

```diff
@@ -376,24 +397,24 @@ The responses looks like:
         "testattr" : "test",
         "portsfile" : "true"
       },
-      "tasks" : {
-        "r1A2WoRbTwKZ516z6NEs5A:36619" : {
-          "node" : "r1A2WoRbTwKZ516z6NEs5A",
-          "id" : 36619,
-          "type" : "transport",
-          "action" : "indices:data/write/reindex",
-          "status" : {    <1>
-            "total" : 6154,
-            "updated" : 3500,
-            "created" : 0,
-            "deleted" : 0,
-            "batches" : 36,
-            "version_conflicts" : 0,
-            "noops" : 0
-          },
-          "description" : ""
-        }
-      }
+      "tasks" : [ {
```

Comment:
Something I just noticed, does this break the format compatibility by instead of return an object using the task id as the key, it returns an array of tasks? If so, we need to mark this as breaking! (and put notes in the migration guide)